notebooks/bar_plots.ipynb

```diff
@@ -5,15 +5,18 @@
    "id": "e6e6b6b0",
```

Comment:
<p>Instead of "methods of the Bokeh <a href="https://docs.bokeh.org/en/latest/docs/reference/plotting/figure.html#bokeh.plotting.figure" target="_blank">Figure</a> class", we can call them "glyphs" to keep the same terminologies we use in the docs. What do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokeh/dataviz-fundamentals/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Alright. Done</p>